.travis.yml

```diff
@@ -1,10 +1,3 @@
-dist: trusty
-sudo: required
-language: go
-
-services:
```

Comment:
is it a good idea? does it mean we expect docker to be in default build os/vm/container?

Reply:
See my last comment at #45.